content/docs/esc/integrations/dynamic-secrets/aws-parameter-store.md

```diff
@@ -0,0 +1,104 @@
+
+* `esc open <your-org>/<your-environment>` command of the [Pulumi ESC CLI](/docs/esc-cli/)
+* `pulumi env open <your-org>/<your-environment>` command of the [Pulumi CLI](/docs/install/)
+
+Make sure to replace `<your-org>` and `<your-environment>` with the values of your Pulumi organization and environment file respectively. You should see output similar to the following:
+
+```json
+{
+  "aws": {
+    "login": {
+      "accessKeyId": "ASIA....",
+      "secretAccessKey": "mWdm....",
+      "sessionToken": "Fwo...."
+    },
+    "params": {
+      "myKey": "latest value",
+      "myKeyByVersion": "old value",
+      "myKeyByVersionLabel": "stable value",
+      "secureKey": "secret value",
+      "myList": [
+        "value 1",
+        "value 2"
+      ]
+    }
+  }
+}
+```
+
+## Inputs
+
+| Property | Type                                       | Description                                                                                                                             |
+|----------|--------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------|
+| `region` | string                                                   | The AWS region to use.                                                                                                    |
+| `login`  | [AWSParameterStoreLogin](#awsparameterstorelogin)        | Credentials to use to log in to AWS.                                                                                      |
+| `get`    | map[string][AWSParameterStoreGet](#awsparameterstoreget) | A map from names to parameters to read from AWS Parameter Store. The outputs will map each name to the parameters's data. |
+
+### AWSParameterStoreLogin
+
+| Property          | Type   | Description                                 |
+|-------------------|--------|---------------------------------------------|
+| `accessKeyId`     | string | The AWS access key ID                       |
+| `secretAccessKey` | string | The AWS secret access key                   |
+| `sessionToken`    | string | [Optional] - The AWS session token, if any. |
+
+### AWSParameterStoreGet
+
+| Property  | Type    | Description                                                                                                                                                |
+|-----------|---------|------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `name`    | string  | The name of the parameter to import. To query by parameter label, use `"name": "name:label"`. To query by parameter version, use `"name": "name:version"`. |
```

Comment:
I know this isn't part of this PR but I do wonder if this should be `id` over `name` since it can be just the name or the name + some metadata

Reply:
that's reasonable - though the aws api also calls it [name](https://docs.aws.amazon.com/systems-manager/latest/APIReference/API_GetParameters.html)